nano/node/node.hpp

```diff
@@ -534,6 +534,7 @@ class node : public std::enable_shared_from_this<nano::node>
 	nano::keypair node_id;
 	nano::block_uniquer block_uniquer;
 	nano::vote_uniquer vote_uniquer;
+	const std::chrono::system_clock::time_point startup_time;
```

Comment:
Use the `std::chrono::steady_clock` to get an accurate time difference, as the `system_clock` can be easily manipulated.